components/o-autocomplete/src/js/autocomplete.js

```diff
@@ -210,6 +212,9 @@ class Autocomplete {
 		if (opts.suggestionTemplate) {
 			this.options.suggestionTemplate = opts.suggestionTemplate;
 		}
+		if (opts.isHighlightCorrespondingToMatch) {
+			this.options.isHighlightCorrespondingToMatch = Boolean(opts.isHighlightCorrespondingToMatch);
+		}
```

Comment:
Wondering if it's simpler to take of Boolean coercion in `Boolean` to set this to false instead of only applying when value is present? It would mean we can remove the guarding on L302. What do you think?

```suggestion
	this.options.isHighlightCorrespondingToMatch = Boolean(opts.isHighlightCorrespondingToMatch);

```